### DISCUSSION TITLE
Changing CartesianChart origin

### DISCUSSION THREAD

**Discussion by DaiYukichi** [patrykandpatrick/vico#773] [Questions]
### Question

Good day,

Our team is using you library to display monitoring values that comes from temperature and humidity sensors. We are looking for a way to customize the CartesianChart so we can change the origin point. We want to do this for managing the TopBottomShader to diplay bottomShader in other point of the chart (specified by the user as the ideal temperature). 

Your help would be appreciable, God bless your team and this project, thank you!

![details _monitor + temp](https://github.com/patrykandpatrick/vico/assets/28677123/e32feeff-5b31-4e00-8a9b-432d9e01766d)

This is our designer example.

### Vico version(s)
2.0.0-alpha21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hi @DaiYukichi, with Vico v2.0.0-alpha.27 you can use `LineCartesianLayer.AreaFill.double` and its `splitY` parameter to change the origin point.
